flattentool/schema.py

```diff
@@ -124,7 +124,22 @@ def parse_schema_dict(self, parent_path, schema_dict, parent_id_fields=None, tit
             parent_path = parent_path + '/'
         parent_id_fields = parent_id_fields or []
         title_lookup = self.title_lookup if title_lookup is None else title_lookup
-        if 'properties' in schema_dict:
+
+        if 'type' in schema_dict and schema_dict['type'] == 'array' \
+                and 'items' in schema_dict and 'oneOf' in schema_dict['items']:
+            for oneOf in schema_dict['items']['oneOf']:
```

Comment:
I think another condition here saying exclude any oneOf that are not of type `object`, as calling `parse_schema_dict` for something that is not an object will raise a confusing warning.

We should probably raise a warning for any use of 'oneOf' or 'anyOf' that is not supported, but that does not have to be in this pull request.